nixos/doc/manual/development/what-happens-during-a-system-switch.chapter.md

```diff
@@ -37,7 +37,7 @@ of actions is always the same:
 - Forget about the failed state of units (`systemctl reset-failed`)
 - Reload systemd (`systemctl daemon-reload`)
 - Reload systemd user instances (`systemctl --user daemon-reload`)
-- Set up tmpfiles (`systemd-tmpfiles --create`)
+- Reactivate sysinit (systemctl restart sysinit-reactivation.target)
```

Comment:
nit: (the rest of these parentheticals are backquoted, might as well stay consistent)

```suggestion
- Reactivate sysinit (`systemctl restart sysinit-reactivation.target`)
```